js/currencycom.js

```diff
@@ -346,19 +361,23 @@ module.exports = class currencycom extends Exchange {
             if (id.indexOf ('/') >= 0) {
                 symbol = id;
```

Comment:
Let's standardize the symbol for swap markets, according to these rules: https://github.com/ccxt/ccxt/wiki/Manual#contract-naming-conventions

So, `GBP/HKD_LEVERAGE > GBP/HKD:XXXXX`, where XXX is the settlement asset (assuming quote asset).